Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/HandConstraint.cs

```diff
@@ -178,28 +193,18 @@ protected IMixedRealityInputSystem InputSystem
             {
                 if (inputSystem == null)
                 {
-                    MixedRealityServiceRegistry.TryGetService<IMixedRealityInputSystem>(out inputSystem);
+                    MixedRealityServiceRegistry.TryGetService(out inputSystem);
```

Comment:
Why remove the Interface?

Reply:
Just for brevity since it can be inferred via the input.

Reply:
huh, and it still works? If no interface is provided then it will be IMixedRealityService which shouldn't be a valid key

Reply:
AFAIK it works since C# will infer generic types based on arguments. Just checked the debugger to make sure:
![image](https://user-images.githubusercontent.com/13305729/62401524-6cd1de00-b538-11e9-9c6e-61f106fd8258.png)

Reply:
Ahhh because of the input parameter. I ran into an issue with this but there was no T serviceInstance to infer from